 <?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReconVizWindow</class>
 <widget class="QWidget" name="ReconVizWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Recon Visualization</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="vizTabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="hostTab">
      <attribute name="title">
       <string>Host Map</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QFrame" name="hostMapFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QWidget" name="hostMapWidget" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="portsTab">
      <attribute name="title">
       <string>Port Distribution</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QFrame" name="portDistFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QWidget" name="portDistWidget" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="subdomainsTab">
      <attribute name="title">
       <string>Subdomains</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QFrame" name="subdomainFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QWidget" name="subdomainWidget" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="summaryTab">
      <attribute name="title">
       <string>Summary</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QFrame" name="summaryFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QWidget" name="summaryWidget" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="controlsGroupBox">
     <property name="title">
      <string>Visualization Controls</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="vizTypeComboBox">
        <item>
         <property name="text">
          <string>Network Graph</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bar Chart</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Pie Chart</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Treemap</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Heatmap</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="dataSelectLabel">
        <property name="text">
         <string>Data:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="dataSelectComboBox">
        <item>
         <property name="text">
          <string>Hosts</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Open Ports</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Services</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Subdomains</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="refreshButton">
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exportButton">
        <property name="text">
         <string>Export</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>